lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -351,7 +383,7 @@ module.exports = {
                 "RequestParameters" : {},
                 "Integration" : {
                   "IntegrationHttpMethod" : "POST",
-                  "Type" : "AWS",
+                  "Type" : "${integrationType}",
```

Comment:
@eahefnawy I've converted to "Objects" here (vs. Strings): https://github.com/serverless/serverless/pull/2262 agreed it's much better.